### PR TITLE
Register skills and interests

### LINKED ISSUE #9: Hacker info input

### PR DESCRIPTION
**Pull request by nbybhen**
Allows for users to be able to add skills to their profile on registration (based on server roles), as well as manually input more skills / interests via modal.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -46,9 +82,44 @@ async def ping(interaction: discord.Interaction):
     name="register", description="Register as a hacker for matchmaking."
 )
 async def register(interaction: discord.Interaction):
+    structure_roles = [
```
Comment (menamerai) [inline bot.py:85]: Before the hackathon start make sure to sweep this again to make sure the roles are updated

File: bot.py [hunk 2]
```diff
@@ -27,6 +27,42 @@
...
+        label="List out your interests.",
+        style=discord.TextStyle.long,
+        placeholder="Type your interests here...",
+        required=True,
+        max_length=300,
+    )
+
+    async def on_submit(self, interaction: discord.Interaction):
+        try:
+            print("Inputted interests: ", self.interests.value)
```
Comment (menamerai) [inline bot.py]: Use `logging.info` instead of print, I've set it up to that logging also prints to the command line

**Review by menamerai — CHANGES_REQUESTED** [review]
Use `logging`, everything else looks good

File: bot.py (hunk 2 above)
Comment (nbybhen) [reply bot.py]: changed

**Pull request merged**